server/entity/src/broadcasters/inventory.rs

```diff
@@ -126,6 +130,66 @@ fn is_equipment_update(held_item: usize, slot: SlotIndex) -> Result<Equipment, (
     }
 }
 
+/// System for damaging inventory items which should take damage.
+#[fecs::event_handler]
+pub fn on_damage_item(event: &ItemDamageEvent, game: &mut Game, world: &mut World) {
```

Comment:
This function is pretty long—it would be good to split it up into separate parts.

Reply:
I pulled out a function for sending the sound effect, so it should be more reasonable now.